CHANGELOG.md

```diff
@@ -6,13 +6,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
-### Added
-
-- Introduce possibility of specifying policy order restrictions in their schemas. APIcast now shows a warning when those restrictions are not respected [#1088](https://github.com/3scale/APIcast/pull/1088), [THREESCALE-2896](https://issues.jboss.org/browse/THREESCALE-2896)
```

Comment:
This is being removed?

Reply:
No, it's a feature that's only on master. The entry in the changelog was introduced by mistake.